src/Servers/IIS/test/IISIntegration.FunctionalTests/IISIntegration.FunctionalTests.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(StandardTestTfms)</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Content Include="AppHostConfig\*.config" CopyToOutputDirectory="PreserveNewest" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Reference Include="Microsoft.AspNetCore.Server.IISIntegration" />
+  </ItemGroup>
+
+  <ItemGroup>
```

Comment:
Low-importance nit: you can condense blocks of the same item type. I think this applies to several .csproj files altered in this PR.